src/Common/Common.Api/Admin/EncryptionKey.cs

```diff
@@ -0,0 +1,37 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+ 
+using System;
+
+namespace Microsoft.PowerBI.Common.Api.Encryption
```

Comment:
Similar comment to the other file. You could create a directory structure for Encryption too but right now it should be Admin based on directory structure

Reply:
Moved to Encryption directory